docs/installationUbuntu20.04.md

```diff
@@ -0,0 +1,169 @@
+# Tested on Ubuntu 18.04.4 LTS (bionic Beaver) and 20.04 LTS (Focal Fossa)
+
+## Prerequisites
+* yarn
+```
+curl -sS https://dl.yarnpkg.com/debian/pubkey.gpg | sudo apt-key add -
+echo "deb https://dl.yarnpkg.com/debian/ stable main" | sudo tee /etc/apt/sources.list.d/yarn.list 
+sudo apt update && sudo apt install yarn
+```
+* Rust
+```
+curl https://sh.rustup.rs -sSf | sh
+```
+* Cargo
+```
+sudo apt install cargo
+```
+
+## Update ubuntu
+```
+sudo apt-get update
+sudo apt-get upgrade
+sudo apt-get dist-upgrade
+```
+
+## Install dependencies
+```
+sudo apt-get install git ssh curl wget gcc g++ cmake \
+libstxxl-dev libxml2-dev libsparsehash-dev libbz2-dev zlib1g-dev libzip-dev libgomp1 \
+pkg-config libgdal-dev libtbb-dev psmisc build-essential postgresql postgis \
+clang libboost-all-dev libexpat1-dev libjsoncpp-dev libncurses5-dev lua5.3 liblua5.3-dev \
+powerline fonts-powerline zsh
+```
+
+## Create sources directory
+```
+cd ~
+mkdir sources
+```
+
+## Install and compile capnproto
+```
+cd sources
+curl -O https://capnproto.org/capnproto-c++-0.8.0.tar.gz
+tar zxf capnproto-c++-0.8.0.tar.gz
+cd capnproto-c++-0.8.0
+./configure
+make -j6 check
+sudo make install
+```
+
+## Generate ssh key
```

Comment:
Why is it in this doc?

Reply:
This is a step by step installation guide that @alexis-viallard  and @kaligrafy made for a brand new install on an ubuntu virtual machine, so it has everything to set up a development environment in ubuntu. That probably should be mentioned somewhere, as simple users do not require all this...

Reply:
Maybe, but not here. Or make a different title for the document. We are not in the business of giving basic sys admin instruction. We could link to a document somewhere. 

We might need it for the lab or some other internal documentation, but it does not belong here.

Reply:
I renamed the document, the title had already been updated. I like to have this step by step and complete documentation for a completely new installs. Our target audience are not all software developers. The newer versions of this PR have notes under each of those title, so reader know if this step is required for their purpose or not.